ui_framework/components/panel/_panel.scss

```diff
@@ -1,11 +1,18 @@
 .kuiPanel {
-  border: $globalBorderThick;
+  border: $globalBorderThin;
+  border-radius: $globalBorderRadius;
 }
 
 .kuiPanel--withHeader {
+  // border-top: none;
+}
+
+.kuiPanel--withToolbar {
```

Comment:
Can we rename this `kuiPanel--withToolBar` to be consistent with the casing of the ToolBar component?